pygsheets/sheet.py

```diff
@@ -18,7 +18,7 @@
 class SheetAPIWrapper(object):
 
     def __init__(self, http, data_path, seconds_per_quota=100, retries=1, logger=logging.getLogger(__name__),
-                 check=True):
+                 check=True, requestBuilder=None):
```

Comment:
please change `requestBuilder` to `request_builder` for consistency.

Reply:
Done.